### ISSUE TITLE
Generate GLB setting is user scoped

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/editor#221]
Not sure if it is by design, but decided to post just in case.

When working with the team, we found that the setting that converts the imported model to GLB is per user and not per project:
![image](https://user-images.githubusercontent.com/5677782/99116655-0ec9ec80-25fd-11eb-9cd1-7f7b54148800.png)

If one user changes the setting, it does not affect the others.

**Comment by Maksims** [comment]
As GLB still not covers 100% of users to switch from FBX, I would be "unhappy" if the setting would switch for me without my explicit action for it.

**Comment by LeXXik** [comment]
Right, on the other hand most of the project settings are project scoped. I think most of the team mates are aware of the fact that changing the project setting would affect everyone. I was not aware that some settings only affect your account, so we got unexpected results.

**Comment by Maksims** [comment]
Pretty much all import settings are user_project scoped.
It is better to challenge those things at the design stage, rather than making switches aftermath, leading to users confusions.

And this only affects all existing projects. While all new projects will have defaults to GLB, correct?

**Comment by yaustar** [comment]
We could add a description in the tooltip or section to make it clear that import settings are user scoped.

LeXXik is an end user so they couldn't challenge it at the design stage 😅

**Comment by willeastcott** [comment]
This is by design and clearly marked in the UI:

<img width="1209" height="345" alt="Image" src="https://github.com/user-attachments/assets/b5b44de2-26e5-4936-b3ad-d141b4903903" />

Closing.
